### PR TITLE
[OJS][main] #7608 Add DC Metadata article view additions test

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

File: cypress/tests/integration/ArticleViewDCMetadata.spec.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+		cy.logout();
+		cy.login('admin', 'admin');
+
+		cy.visit('index.php/publicknowledge/manageIssues#future');
+		cy.get('span:contains("' + issueTitle + '")').prev('a.show_extras').click();
+		cy.get('tr:contains("' + issueTitle + '")').next().contains('a', 'Publish Issue').click();
+		cy.get('input[id="sendIssueNotification"]').click();
+		cy.get('button[id^=submitFormButton]').click();
+
+		cy.logout();
```
Comment (bozana) [inline cypress/tests/integration/ArticleViewDCMetadata.spec.js:60]: This does not have consequences for other articles, right?
Would be maybe be better to unpublish/re-publish just that submission? 
:thinking:

File: cypress/tests/integration/ArticleViewDCMetadata.spec.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+		cy.get('meta[name="DC.Identifier.URI"]')
+			.should("have.attr", "content", "http://localhost:8000/index.php/publicknowledge/article/view/mwandenga-signalling-theory");
+
+		cy.get('meta[name="DC.Language"]')
+			.should("have.attr", "content", "en")
+			.and("have.attr", "scheme", "ISO639-1");
+
+		cy.get('meta[name="DC.Rights"]')
+			.should("have.attr", "content", "Copyright (c) 2022 Journal of Public Knowledge");
```
Comment (bozana) [inline cypress/tests/integration/ArticleViewDCMetadata.spec.js:126]: Should the year be adapted in the same way as date above?

**Review by bozana — COMMENTED** [review]
Hi @defstat, only two comments :-)
Thanks a lot!

File: cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+						},
+					],
+				}
+			],
+			source: {
+				name: 'Journal of Public Knowledge',
+				issn: '0378-5955',
+				issue: '2',
+				volume: '1',
+				uri: 'http://localhost/index.php/publicknowledge',
```
Comment (bozana) [inline cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js:107]: Could we here use Cypress.env('baseUrl') ?

File: cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js [hunk 4]
```diff
@@ -0,0 +1,496 @@
...
+				issueTitle: 'Vol. 1 No. 2 (2014)',
+				doiPrefix: '10.1234',
+			},
+			identifiers: {
+				pageNumber: '71-98',
+				uri: '',
+				doi: '',
+			},
+			urlPath: 'testing-dc-metadata-submission-' + uniqueId,
+			licenceUrl: 'https://www.google.com',
```
Comment (bozana) [inline cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js:117]: Could we have rather use https://creativecommons.org/licenses/by/4.0/ :-DDD ?

File: cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js [hunk 5]
```diff
@@ -0,0 +1,496 @@
...
+				{
+					label: 'PDF',
+					genre: 'Article Text',
+					mimeType: 'application/pdf',
+				}
+			],
+		};
+
+		dcElements = {
+			localised: [
```
Comment (bozana) [inline cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js:131]: I think we should use 'localized' -- the American English version :-)

**Review by bozana — COMMENTED** [review]
@defstat, great! Just a few questions :-)
I think you do not need to submit your pkp-lib submodule because it contains no changes...

File: cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js (hunk 3 above)
Comment (defstat) [reply cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js:107]: Sure. Fixed.

File: cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js (hunk 4 above)
Comment (defstat) [reply cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js:117]: Much better for sure. Fixed.

File: cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js (hunk 5 above)
Comment (defstat) [reply cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js:131]: Fixed.

**Pull request merged**